<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineColPosWidget</class>
 <widget class="QWidget" name="LineColPosWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lbl_line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Line:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_col">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Col:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_pos">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Pos:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
